PrestaShop/PrestaShop
# First part of getting rid of the legacy layout for migrated pages

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop
| Description?  | This PR is the first part to get rid of the legacy layout on migrated pages.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | Partially #12925
| How to test?  | See below

## How to test

First, you should set the constant `_PS_USE_NEW_LAYOUT` to `true` in `/admin-dev/index.php`.
Then, you can mostly test this PR by comparing side to side the following parts with the develop branch:

- You should have the exact same design on the BO (especially on migrated pages)
- The breadcrumbs should be identical
- The toolbar title should be identical
- Hooks should be working the same (especially display hooks)
- You should be able to Login/Logout
- When choosing **not** to remember your login credentials, you should be logged out after a period of non-activity.

What is not working in this PR and will come in another one is : 

- The breadcrumbs filters
- The toolbar buttons (Help links should be working though)

Those are points we should focus on, but this PR has a global impact on the BO and exploratory tests should be done too (mostly on migrated pages).





---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/prestashop/prestashop/21469)
